google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -689,7 +689,7 @@ public long read(
     Span span = startSpan(HttpStorageRpcSpans.SPAN_NAME_READ);
     Scope scope = tracer.withSpan(span);
     try {
-      checkArgument(position >= 0, "Position should be non-negative, is %d", position);
+      checkArgument(position >= 0, "Position should be non-negative, is %s", position);
```

Comment:
I can't tell the difference between these two. %d looks correct to me. How about we use string concatenation with a plus sign here instead to avoid the issue.

I do notice this should probably not be inside the try block.

Reply:
%d had been correct with standard formater, but is incorrect when using guava's Precondition library formatting. (You can check documentation and code linked in the associated issue)

Reply:
The mere fact that we have to check documentation and that this is not obvious strongly suggests that we shouldn't use a format string at all. Just use string concatenation and call it a day.

Reply:
Done. I have used string concatenation `+` instead of `%s`. @elharo PTAL